ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/StreamInfoImpl.java

```diff
@@ -41,8 +51,18 @@ public String getTopic() {
   }
 
   @Override
-  public String getFormat() {
```

Comment:
This is a public API. Should we leave it here and mark as deprecated, to be removed in a future release?

Reply:
My personal view is no, I think the Java client is new enough we can get away with just changing this. People will get a compilation error and will just switch to the new method. But... we don't actually have a defined policy for this yet.

Reply:
Meh... done.